celeritas.spec

```
@@ -1,7 +1,5 @@
-### RPM external celeritas v0.6.0
-%define celeritas_gitversion %(echo %{realversion} | sed -e 's|^v||;s|-.*||')
-%define tag dfa4cde7d7d65bf656b17a24c59fcc030aa6b0d9 
-Source: git+https://github.com/celeritas-project/celeritas?obj=develop/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
+### RPM external celeritas 0.6.3
+Source: https://github.com/celeritas-project/celeritas/archive/refs/tags/v%{realversion}.tar.gz
```

Can you please use

which is hashed, released as an immutable artifact, and is guaranteed to have the correct versioning information embedded (not relying on the `GIT_DESCRIBE` hack below)?
```suggestion
Source: https://github.com/celeritas-project/celeritas/releases/download/v%{realversion}/celeritas-%{realversion}.tar.gz
```